packages/data/src/reviews/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+
+import { registerStore, select } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_NAME } from './constants';
+import * as selectors from './selectors';
+import * as actions from './actions';
+import * as resolvers from './resolvers';
+import controls from './controls';
+import reducer from './reducer';
+
+const storeSelectors = select( STORE_NAME );
+
+// @todo This is used to prevent double registration of the store due to webpack chunks.
```

Comment:
Is there a github issue somewhere for this? It would be great to be able to track when we can remove this conditional by adding a link in the comment if possible.

Reply:
There is. I added it to the comment here and other instances in https://github.com/woocommerce/woocommerce-admin/pull/4941/commits/93cfb484fa5ff7beda8be30d1b124298daf3ac1d